Frameworks/CoreText/CTRun.mm

```diff
@@ -256,11 +261,15 @@ void CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange) {
         range.length = [string length];
     }
 
-    int numGlyphs = curRun->_characters.size();
+    int numGlyphs = [string length];
```

Comment:
>numGlyphs [](start = 8, length = 9)

This won't be true for handling ligatures or other instances where there's not a 1-1 matching between a and a glyph